### PR TITLE
Update gitignore and prettify some files

### PR DESCRIPTION
**Pull request by aruppert**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
-**/node_modules
```
Comment (tobiaserich) [inline .gitignore]: clever solution 👍

File: .gitignore [hunk 2]
```diff
@@ -23,4 +23,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 brainstorming.md
-storybook-static
\ No newline at end of file
+storybook-static
```
Comment (tobiaserich) [inline .gitignore]: Why?

Comment (aruppert) [reply .gitignore]: Outdated

File: .gitignore (hunk 1 above)
Comment (aruppert) [reply .gitignore]: Yeah, no idea who told me thaat!

**Pull request merged**